gnovm/pkg/gnolang/machine.go

```diff
@@ -25,12 +25,23 @@ type Exception struct {
 	// Frame is used to reference the frame a panic occurred in so that recover() knows if the
 	// currently executing deferred function is able to recover from the panic.
 	Frame *Frame
+
+	Stacktrace Stacktrace
 }
 
 func (e Exception) Sprint(m *Machine) string {
 	return e.Value.Sprint(m)
 }
 
+// RealmUnhandledPanicException represents an error thrown when a panic is not handled in the realm.
+type RealmUnhandledPanicException struct {
+	Descriptor string // Description of the unhandled panic.
+}
+
+func (e RealmUnhandledPanicException) String() string {
```

Comment:
(1) the panic could be called and recovered by a package, so "realm" is pointless; (2) "exception" is redundant with panic, and refers to an internal data structure.

Better `UnhandledPanicError`, and rename the method to `Error() string`; so it implements the `error` interface.

Reply:
done here: https://github.com/gnolang/gno/pull/2145/commits/778965e4d5c91ba6ce4c8d754dbe5d1cf4a89751